app/services/payout/paypal_service.rb

```diff
@@ -0,0 +1,73 @@
+module Payout
+  class PaypalService < Service
+    def perform
+      return if skip_publisher?
+      return if !@publisher.has_verified_channel? || @publisher.locked? || @publisher.excluded_from_payout? || @publisher.hold? || @publisher.wire_only?
```

Comment:
I think we can remove this line

Reply:
https://github.com/brave-intl/publishers/pull/2859/commits/85d49ba9d79a948fb7f6c818f2875fb61a378f8f